libs/addData.js

```diff
@@ -4,18 +4,19 @@
  * @module addData
  */
 const bluebird = require('bluebird')
+const omit = require('lodash/omit')
 const log = require('./logger').namespace('addData')
 
 module.exports = (entries, doctype) => {
   const cozy = require('./cozyclient')
-  return bluebird.mapSeries(entries, entry => {
+  return bluebird.mapSeries(entries, async entry => {
     log('debug', entry, 'Adding this entry')
-    return cozy.data.create(doctype, entry)
-    .then(dbEntry => {
-      // also update the original entry _id to allow saveBills' linkBankOperation entries to have
-      // an id
-      entry._id = dbEntry._id
-      return dbEntry
-    })
+    const dbEntry = await (entry._id
+      ? cozy.data.update(doctype, entry, omit(entry, '_rev'))
```

Comment:
If this function now does some updates, its name does not fit. Plus, we also have a createOrUpdate function.

Reply:
Where is this function ?

https://github.com/cozy/cozy-konnector-libs/search?utf8=%E2%9C%93&q=createOrUpdate&type=

What do you think if we rename this function to "createOrUpdateEntries" ?